ccd-definition/FixedLists/UnemployedRadioFixedListLRspec.json

```diff
@@ -0,0 +1,20 @@
+[
```

Comment:
Instead of Radio and FixedList, which are reference to field types/controls, I would name it something like UnemploymentType

Reply:
this is helping me with readability have a look at this complex type

[
  {
    "ID": "EmploymentTypeComplexTypeDQLRspec",
    "ListElementCode": "employmentTypeComplexTypeDQLRspec",
    "FieldType": "MultiSelectList",
    "FieldTypeParameter": "EmploymentTypeCheckboxFixedListLRspec",
    "ElementLabel": "Employed or self-employed?",
    "HintText": "Select both if your client is employed and self-employed.",
    "SecurityClassification": "Public"
  }
]
We can discuss naming convention